cloudera/hue
# [backend/core] remove weaker cipher suites from defaults

We are removing ECDHE-ECDSA-CHACHA20-POLY1305 specifically. Few other
suites are also removed as those aren't supported by gunicorn anyways.

Testing:
1. Run the following command to find the list of supported ciphers
   `nmap -sV --script +ssl-enum-ciphers -p $PORT $HOST`
2. Run the same command after the applying the change. You should find
   `TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256` out of the supported
   list now.
